src/gnssro/gnssro_bufr2ioda.py

```diff
@@ -110,11 +111,15 @@ def main(args):
     VarAttrs[('longitude', 'MetaData')]['_FillValue'] = float_missing_value
     VarAttrs[('height', 'MetaData')]['_FillValue'] = float_missing_value
 
+    if addLSW:
+        VarAttrs[('localSpectralWidth', 'ObsError')]['units'] = 'Percentage'
```

Comment:
If I understand correctly, this will be localSpectralWidth@ObsError. That will be interpreted as the obs error of localSpectralWidth?

Reply:
What would you say is the best location for this variable @hailingz? ObsValue? MetaData? @BenjaminRuston @huishao-r which is the best place for it?

Reply:
This is a hard one to decide. LSW actually is considered as QC flags in the latest discussion of the new bufr table, instead of putting it under observation error section. What do we have under preQC currently?

Reply:
In the IODA world, isn't it more like a `MetaData` variable (instead of `ObsVaue`, `ObsError`, `PreQC`)? With that saved as a `MetaData` variable, it can be used later in QC procedures to derive other quantities. If the `PreQC` route is chosen, it implies that a decision will be needed on deciding which classes are going to be considered as "good" (remember, `PreQC` is an integer-based variable). If it's kept as a float in the `MetaData` group, then the user is free to decide what is considered "good" later on when using the file.